src/protocols/secure_channel/CASESession.cpp

```diff
@@ -214,23 +232,22 @@ CHIP_ERROR CASESession::Init(uint16_t myKeyId, SessionEstablishmentDelegate * de
 }
 
 CHIP_ERROR
-CASESession::ListenForSessionEstablishment(uint16_t myKeyId, Transport::FabricTable * fabrics,
+CASESession::ListenForSessionEstablishment(uint16_t mySessionId, Transport::FabricTable * fabrics,
```

Comment:
Same here, local session ID.

However, I don't think we can allocate "one" session ID, otherwise we need one separate call of ListenForSessionEstablishment every time we "eventually" expect a CASE session. Overall, not in this PR, I would recommend picking the local session ID when Sigma2 is received.